sample/SampleHost/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "ConnectionStrings": {
+        "AzureWebJobsDashboard": "UseDevelopmentStorage=true",
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true"
```

Comment:
@mamaso updated setting values and also rebased from dev to get it up to date with source